impeller/compiler/compiler_unittests.cc

```diff
@@ -90,6 +90,41 @@ TEST_P(CompilerTest, BindingBaseForFragShader) {
   ASSERT_GT(frag_uniform_binding, vert_uniform_binding);
 }
 
+TEST_P(CompilerTest, UniformsHaveBindingAndSet) {
+  if (GetParam() == TargetPlatform::kFlutterSPIRV) {
+    // This is a failure of reflection which this target doesn't perform.
+    GTEST_SKIP();
+  }
+
+#ifndef IMPELLER_ENABLE_VULKAN
+  GTEST_SKIP();
```

Comment:
This should work on all backends. Specifying the binding and set should not be a Vulkan only thing.

Reply:
good catch!